roles/grafana/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Manage rpm repositories
+  import_tasks: redhat.yml
+  when: ansible_os_family == 'RedHat'
+
+- name: Manage deb repositories
+  import_tasks: debian.yml
+  when: ansible_os_family == 'Debian'
+
+- name: Install Grafana
+  package:
+    name: grafana
+
+- name: Place Grafana ini file
+  template:
+    src: grafana.ini.j2
+    dest: /etc/grafana/grafana.ini
+    backup: true
+    owner: root
+    group: grafana
+    mode: 0640
+  notify:
+    - Restart Grafana
+
+- name: Place datasource file
+  template:
+    src: datasources.yml.j2
+    dest: /etc/grafana/provisioning/datasources/datasources.yml
+    owner: root
+    group: grafana
+    mode: 0640
+  notify:
+   - Restart Grafana
```

Comment:
If you could fix the indentation here (needs one `' '` more), the pipeline would be green.

Reply:
Thanks, i changed the code (on more ' ' in line 33).